### DISCUSSION TITLE
Will micronaut-serialization serve as the default json serialization engine in the future?

### DISCUSSION THREAD

**Discussion by chenjpu** [micronaut-projects/micronaut-core#7134] [Q&A] [answered]
**Will micronaut-serialization serve as the default json serialization engine in the future?**
**Will the jackson-core and jackson-databind be removed from the micronaut-core？**

**Comment by graemerocher** [comment, marked as answer]
Yes, though we don't plan to deprecate Jackson, it will always be supported as a second option
